client/src/ui/Form.jsx

```diff
@@ -556,6 +676,44 @@ export default class Form extends Component {
               </div>
             </div>
 
+            <div className="row">
+              <div className="col-md-3 col-sm-3 col-xs-3 form-bottom-row-spacing">
```

Comment:
we upgraded to Bootstrap 4 in an earlier PR, so `col-xs-3` should just be `col-3` (you might have to delete `node_modules` and run `yarn` again)

also, if you're applying the same column width to all screen sizes, you can just have `col-3`: https://getbootstrap.com/docs/4.0/layout/grid/#mix-and-match

```suggestion
              <div className="col-3 form-bottom-row-spacing">
```